### PR TITLE
Add wizard helper

### PR DESCRIPTION
**Pull request by paulrobertlloyd**
This PR adds:

* Wizard helper functions
* Example wizard pages and routes
* Documentation placeholder.

Now needs @fofr to document the magic!

### CODE REVIEW

File: app/views/examples/wizard/where-do-you-live.njk [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+{% block pageNavigation %}
+  {{ govukBackLink({
+    href: paths.back
+  }) }}
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <form method="post" action="{{ paths.current }}" novalidate>
```
Comment (fofr) [inline app/views/examples/wizard/where-do-you-live.njk:14]: You didn't want to include the form layout which abstracts away the paths logic?

File: package/docs/using-data/2. wizard.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+Each fork accepts the following values:
+
+| Name | Type | Description |
+| :--- | :--- | :---------- |
+| **currentPath** | string | **Required.** Path to fork on. |
+| **forkPath** | string/Function | **Required.** Path to fork to based on answer. |
+| **storedData** | string/Array | **Required.** Data to check values against. |
+| **values** | string/Array | Values to include. |
+| **includedValues** | string/Array | Values to exclude. |
```
Comment (fofr) [inline package/docs/using-data/2. wizard.md:51]:
```suggestion
| **includedValues** | string/Array | Values to include. |
| **excludedValues** | string/Array | Values to exclude. |
```

Comment (fofr) [inline package/docs/using-data/2. wizard.md:51]:
```suggestion
| **excludedValues** | string/Array | Values to exclude. |
```

File: app/views/examples/wizard/where-do-you-live.njk (hunk 1 above)
Comment (paulrobertlloyd) [reply app/views/examples/wizard/where-do-you-live.njk:14]: I’m wondering if such layouts should be a different feature (I’ve created page, question and check answer layouts on our current prototype). The important thing the wizard provides is `path.*` values. How you use them is up to you.

File: package/docs/using-data/2. wizard.md (hunk 2 above)
Comment (paulrobertlloyd) [reply package/docs/using-data/2. wizard.md:51]: Oooh, I think I changed this value on our prototype – will reinstate your original name.

File: app/views/examples/wizard/where-do-you-live.njk (hunk 1 above)
Comment (fofr) [reply app/views/examples/wizard/where-do-you-live.njk:14]: Yeah, but if the form doesn't post back to itself the fork logic won't execute

Comment (paulrobertlloyd) [reply app/views/examples/wizard/where-do-you-live.njk:14]: Not sure I understand. The examples include the forms posting to `paths.current` (arguably more visible in a template than an abstracted layout), and even if you don’t do this, isn’t the default behaviour to post to the current page anyway?

**Pull request merged**